### PR TITLE
Update WHSSDatlas_v4.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld [hunk 1]
```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: this needs to be:
```
"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},
```

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld [hunk 2]
```diff
@@ -692,20 +692,25 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Kleven et al. 2023.",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: The publications should be stated as full citations. you don't want to force people to look it up themselves.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld [hunk 3]
```diff
@@ -692,20 +692,25 @@
...
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
+	"https://doi.org/10.1016/j.neuroimage.2014.04.001",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld:709]: indentation missing for list item (two more spaces per row)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld [hunk 4]
```diff
@@ -692,20 +692,25 @@
...
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld:708]:
```
"relatedPublication": [
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.10.017"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.12.080"},
	{"@id": "https://doi.org/10.1016/j.neuroimage.2019.05.016"}  
],
```

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld (hunk 1 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld (hunk 2 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: Full citation has been added.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld (hunk 4 above)
Comment (lzehl) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld:708]: please correct the indentation (2spaces more for each list item)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld [hunk 5]
```diff
@@ -692,20 +692,25 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v4 (Heidi Kleven, Ingvild Bjerke, Francisco Clascá et al. Waxholm Space atlas of the rat brain: A 3D atlas supporting data analysis and integration, 07 February 2023, PREPRINT (Version 1) available at Research Square DOI: 10.21203/rs.3.rs-2466303/v1).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: @aeidi89 thanks. I suggest to define this as rst-compatible markdown:

`"Please refer to the atlas by its RRID: SCR_017124, and cite the following publications:\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: https://doi.org/10.1016/j.neuroimage.2014.04.001\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2015) Addendum to "Waxholm Space atlas of the Sprague Dawley rat brain" [NeuroImage 97 (2014) 374-386]. NeuroImage 105, 561–562. DOI: https://doi.org/10.1016/j.neuroimage.2014.10.017"`

(\n means newline)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld (hunk 4 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld:708]: Thanks @lzehl 
Fixed now.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld (hunk 5 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v4.jsonld]: Thanks @lzehl 
Fixed now.

**Review by lzehl — APPROVED** [review]

**Pull request merged**